packages/react-components/src/InputRpc/CennznetJsonRpc.ts

```diff
@@ -0,0 +1,40 @@
+import defaultJsonRpc from '@polkadot/jsonrpc';
+import createMethod from '@polkadot/jsonrpc/create/method';
+import createParam from '@polkadot/jsonrpc/create/param';
+import  userBare  from '@cennznet/api/rpc';
+
+const userRpc = Object.entries(userBare).reduce((user, [sectionName, methods]) => {
+    // @ts-ignore
+    user[sectionName] =  { methods: methods.reduce((section, def) => {
+        const {
+            description = 'User defined',
```

Comment:
wondering how this description field is used?

Reply:
This is default.. If nothings is specified then this will be used..